Library/Formula/h2o.rb

```diff
@@ -30,7 +30,10 @@ def install
     mkdir_p etc/"h2o"
     mkdir_p var/"h2o"
     (var+"h2o").install "examples/doc_root/index.html"
-    (etc+"h2o/h2o.conf").write conf_example
+    # RuntimeError: Will not overwrite /usr/local/etc/h2o/h2o.conf
+    # We could just remove it with rm_f, but the hope is users actively
+    # replace the conf themselves with something less limited.
+    (etc+"h2o/h2o.conf").write conf_example unless (etc+"h2o/h2o.conf").exist?
```

Comment:
Write to the `buildpath` and use `etc.install` which will handle the `.exist?` situation. Thanks!